yarn.lock

```diff
@@ -10810,9 +10878,9 @@ secp256k1@^4.0.2:
     node-addon-api "^2.0.0"
     node-gyp-build "^4.2.0"
 
-secretjs@^0.17.0:
+secretjs@0.17.5, secretjs@^0.17.0:
```

Comment:
do we actually use `secretjs`?

Reply:
The `walletconnect-keplr` client does :/